doc/stdenv.xml

```diff
@@ -222,9 +222,10 @@ genericBuild
    </footnote>
    But even if one is not cross compiling, the platforms imply whether or not
    the dependency is needed at run-time or build-time, a concept that makes
-   perfect sense outside of cross compilation. For now, the run-time/build-time
-   distinction is just a hint for mental clarity, but in the future it perhaps
-   could be enforced.
+   perfect sense outside of cross compilation. By default, the
+   run-time/build-time distinction is just a hint for mental clarity, but with
+   <varname>strictDeps</varname> set it is somewhat enforced even in the native
```

Comment:
```suggestion
   <varname>strictDeps</varname> set it is enforced even in the native
```

Reply:
Well it's not *complete* enforcement. Maybe "mostly"?

Reply:
```suggestion
   <varname>strictDeps</varname> set it is mostly enforced even in the native
```